muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/feed/FeedRESTService.java

```diff
@@ -0,0 +1,69 @@
+package fi.otavanopisto.muikku.plugins.feed;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.ejb.Stateful;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import fi.otavanopisto.muikku.plugin.PluginRESTService;
+import fi.otavanopisto.muikku.plugins.feed.model.Feed;
+import fi.otavanopisto.muikku.plugins.feed.model.FeedItem;
+import fi.otavanopisto.security.rest.RESTPermit;
+import fi.otavanopisto.security.rest.RESTPermit.Handling;
+
+@Path("/feed")
+@RequestScoped
+@Stateful
+@Produces ("application/json")
+public class FeedRESTService extends PluginRESTService {
+
+  private static final long serialVersionUID = -10681497398136513L;
+
+  @Inject
+  private FeedDAO feedDao;
```

Comment:
Might as well capitalize DAO in variable name to maintain naming consistency across the application